### PR TITLE
Wrapper for traits instead of Stub

### PR DESCRIPTION
**Pull request by xgreenx**
Removed stub implementation. Instead `trait_definition` generates an implementation of builder for `AccountId`. Added `brush::wrapper` macro to generate alias for wrapper explicitly.

Speed up the build of tests.
Updated tests of reetrancy_guard.
Removed `blake2b_256_as_u32` and `blake2b_256`, instead using analog from ink!=)
Added `#[ink(selector = X)]` in trait by default to be sure that it is right.
Updated readme

### CODE REVIEW

**Review by coreggon11 — APPROVED** [review]
I like it

File: tests/reentrancy-guard.tests.ts [hunk 1]
```diff
@@ -36,6 +36,21 @@ describe('REENTRANCY_GUARD', () => {
...
 
+  it('Flip on target works', async () => {
+    const { query, contract } = await setup()
+
+    const { tx } = await setup_flip_on_me()
+
+    // Arrange - Ensure flip value is false
+    await expect(query.getValue()).to.have.output(false)
+
+    // Assert
```
Comment (VargSupercolony) [inline tests/reentrancy-guard.tests.ts:47]:
```suggestion
    // Act
```

File: tests/reentrancy-guard.tests.ts [hunk 2]
```diff
@@ -36,6 +36,21 @@ describe('REENTRANCY_GUARD', () => {
...
+    await expect(tx.flipOnTarget(contract.address)).to.eventually.be.fulfilled
+
+    // Assert - Value still must be true
+    await expect(query.getValue()).to.have.output(true)
+  })
+
   it('Call flip on me must fails', async () => {
```
Comment (VargSupercolony) [inline tests/reentrancy-guard.tests.ts:54]:
```suggestion
  it('Call flip on me must fail', async () => {
```

File: utils/brush/lib.rs [hunk 3]
```diff
@@ -5,11 +5,10 @@ pub mod traits;
 
 // Each proc macro here is doing some kind of dark magic, be careful, you can summon devil=)
```
Comment (VargSupercolony) [inline utils/brush/lib.rs:6]:
```suggestion
// Each proc macro here is doing some kind of dark magic, be careful, you can summon the devil =)
```

**Review by VargSupercolony — APPROVED** [review]
add wrapper logic + implementations for all of the contracts

**Review by coreggon11 — APPROVED** [review]

**Pull request merged**